libbeat/cfgfile/cfgfile.go

```diff
@@ -138,6 +139,23 @@ func Load(path string) (*common.Config, error) {
 	return config, nil
 }
 
+// LoadConfigs loads a list of configs data from the given file
+func LoadList(file string) ([]*common.Config, error) {
+	logp.Debug("cfgfile", "Load config from file: %s", file)
+	rawConfig, err := common.LoadFile(file)
+	if err != nil {
+		return nil, fmt.Errorf("Invalid config: %s", err)
```

Comment:
Should we not capitalize the error strings? (https://github.com/golang/go/wiki/CodeReviewComments#error-strings)